Introduction to MongoDB: Give definition for acronym (#27613)
TheOdinProject/curriculum: nodeJS/mongoDB/intro_to_mongoDB.md

### Introduction

> Data persistence is integral to the vast majority of web applications. Local storage has limitations and poses security risks, and SaaS (software as a service) options such as Firebase work well, but ultimately being able to understand how to structure, build, and query your own database are important skills for any full stack developer to have. 

Reviewer: ```suggestion
Data persistence is integral to the vast majority of web applications. Local storage has limitations and poses security risks, and Backend-as-a-Service (BaaS) options such as Firebase work well, ultimately being able to understand how to structure, build, and query your own database are important skills for any full stack developer to have. 
```
I think I'd prefer to use the BaaS term instead, but agree about the acronym definition in this file. Threw in a small grammar fix too.

For example, consider X (formerly known as Twitter). Where are your username and password stored? Or perhaps where all of your posts might be fetched from? How does the application determine what posts are shown to you every time you log on? These operations are handled by a database. 

> Databases are split into "relational" and "non-relational" types of databases, and each handles data and scaling in different manners. [This article](https://circleci.com/blog/SQL-vs-NoSQL-databases/) goes in depth with the difference between the two. This lesson will teach you about MongoDB, a popular NoSQL database.

Reviewer: ```suggestion
Databases are split into "relational" and "non-relational" types of databases, and each handles data and scaling in different manners. Read this article to learn more about the [differences between relational and non-relational databases](https://circleci.com/blog/SQL-vs-NoSQL-databases/).

In this lesson, we will teach you about MongoDB, a popular non-relational database.
```
Mind making this change too while we're at it? Both to fix the link issue but also to be more consistent with terminology. The last sentence has been given its own line for clarity as well.

Author: I think now it now it would be better to move that link to assignment, what do you think?

Author: > Mind making this change too while we're at it? Both to fix the link issue but also to be more consistent with terminology. The last sentence has been given its own line for clarity as well.

Also, I think it will slightly improve the readability to remove the `Read this article to` bit

Learn more about the [differences between relational and non-relational databases](https://circleci.com/blog/SQL-vs-NoSQL-databases/).

### Introduction

> Data persistence is integral to the vast majority of web applications. Local storage has limitations and poses security risks, and Backend-as-a-Service (BaaS) options such as Firebase work well, ultimately being able to understand how to structure, build, and query your own database are important skills for any full-stack developer to have.

Reviewer: ```suggestion
Data persistence is integral to the vast majority of web applications. Local storage has limitations and poses security risks, and while Backend-as-a-Service (BaaS) options such as Firebase work well, ultimately being able to understand how to structure, build, and query your own database are important skills for any full-stack developer to have.
```
Was supposed to add this in the previous review suggestion but completely missed it...oops.

For example, consider X (formerly known as Twitter). Where are your username and password stored? Or perhaps where all of your posts might be fetched from? How does the application determine what posts are shown to you every time you log on? These operations are handled by a database.

> Databases are split into "relational" and "non-relational" types of databases, and each handles data and scaling in different manners.
> 
> In this lesson, you will complete a course that teaches you about MongoDB, a popular non-relational database.

Reviewer: ```suggestion
Databases are split into two types - "relational" and "non-relational", and each type handles data and scaling in different manners. In this lesson, you will complete a course that teaches you about MongoDB, a popular non-relational database.
```
I like moving the article to the assignment section - great change.
With that change, combining these two separate parts makes sense again. A little wording suggestion as well just to improve the sentence flow.

Author: Nice!
